### ISSUE TITLE
Use torch.nn.utils.rnn.pad_sequence instead of manual padding

### ISSUE THREAD

**Issue by kirilklein** [FGA-DIKU/BONSAI#73]
https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_sequence.html
